Formula/swagger2markup-cli.rb

```diff
@@ -8,7 +8,6 @@ class Swagger2markupCli < Formula
 
   bottle do
     rebuild 1
-    sha256 cellar: :any_skip_relocation, x86_64_linux: "021cfe09374afb56b84233fa154091cb7178b45d0c07ed5b4f90c2c8ab1f6ab2"
     sha256 cellar: :any_skip_relocation, all:          "021cfe09374afb56b84233fa154091cb7178b45d0c07ed5b4f90c2c8ab1f6ab2"
```

Comment:
```suggestion
    sha256 cellar: :any_skip_relocation, all: "021cfe09374afb56b84233fa154091cb7178b45d0c07ed5b4f90c2c8ab1f6ab2"
```
This will fix your CI failure.

Reply:
woups